reana_job_controller/config.py

```diff
@@ -91,3 +91,8 @@
 
 IMAGE_PULL_SECRETS = os.getenv("IMAGE_PULL_SECRETS", "").split(",")
 """Docker image pull secrets which allow the usage of private images."""
+
+K8S_STATE_REASON_TO_HUMAN_READABLE = {
+    "OOMKilled": "Job exceeded memory limit.",
```

Comment:
Ideally, we should show what is the memory limit. I didn't add this now for two reasons:
1. Not clear which way to go:
    - a) Kubernetes API call: for that, we would need to change the [roles](https://github.com/reanahub/reana/blob/master/helm/reana/templates/roles.yaml) in `reanahub/reana` to allow RJC to make a REST call and discover the limit, and could be done now in `maint-0.7`.
    - b) Central REANA-Commons config: for that, we would need to do changes in RWC (usual issue with passing config between components https://github.com/reanahub/reana/issues/277), which would be too much for `maint-0.7`?
    
    Note: (b) would be a natural way, which would be in concordance with the decisions taken in https://github.com/reanahub/reana-server/pull/341 about having our DB/config as a source of truth and not rely on Kubernetes API.
2. We could do it together with the user settable limits? So we would either show the user set limit if exists otherwise the namespace default one.

I think that we should do (1.b.) which would naturally go together with (2.) in `0.8.x`.